### ISSUE TITLE
Right click doesn't show menu

### ISSUE THREAD

**Issue by Firefox289** [cheesecakeufo/komorebi#260]
Hey uhhm someone help i downloaded komorebi just for live wallpapers but then i right click on the desktop the menu wont show up pls help

**Comment by praharshbhatt** [comment]
Have the same issue

**Comment by Ev1lbl0w** [comment]
Are you sure komorebi is actually running? komorebi modifies GNOME's settings, such as the wallpaper, and makes it look like it runs on the background, even though it isn't. Could you post the output of komorebi in the terminal when you try to run it, please?

**Comment by praharshbhatt** [comment]
@Ev1lbl0w You're right, it wasnt running earlier. Just the wallpaper was changed (static) and so gave the impression it's running.
I changed a setting in my NVIDIA X settings, restarted, and now it works.

**Comment by Firefox289** [comment]
well if its not running than what do i do to run it?

**Comment by Ev1lbl0w** [comment]
> well if its not running than what do i do to run it?

Just run `komorebi` in the terminal, or open it in your Application Launcher.

We have developed a [fork](https://github.com/Komorebi-Fork/komorebi) that, among may bugfixes, allows to set it up to launch automatically on boot.

**Comment by Firefox289** [comment]
can you give me the command to open it in terminal? im new to linux

**Comment by Ev1lbl0w** [comment]
> can you give me the command to open it in terminal? im new to linux

Yes, it's just `komorebi` :slightly_smiling_face:

**Comment by Firefox289** [comment]
command not found

**Comment by Firefox289** [comment]
well opening it in launcher is not working thats for sure

**Comment by Ev1lbl0w** [comment]
Huum, seems like you need to reinstall it.

**Comment by Firefox289** [comment]
ok i tried it a million times but ok

**Comment by Firefox289** [comment]
not working

**Comment by Firefox289** [comment]
![Screenshot from 2021-02-17 20-54-12](https://user-images.githubusercontent.com/77389527/108207171-4a55e480-711f-11eb-80d0-87a671c7d7be.png)
its like this

**Comment by Ev1lbl0w** [comment]
How are you installing it? Are you using the `.deb` package?

It it's not working in any way, try using our [fork](https://github.com/Komorebi-Fork/komorebi/releases/tag/v2.2.1). If it still doesn't work, please explain every step you're doing to install.

**Comment by Firefox289** [comment]
im using deb

**Comment by Firefox289** [comment]
I feel dumb ._.

**Comment by Firefox289** [comment]
maybe it just need update?

**Comment by Firefox289** [comment]
my screen turned black

**Comment by Firefox289** [comment]
![Screenshot from 2021-02-17 20-59-50](https://user-images.githubusercontent.com/77389527/108207750-129b6c80-7120-11eb-8359-5faa7b5620e9.png)
this happend after i clicked on komorebi

**Comment by Firefox289** [comment]
https://github.com/Komorebi-Fork/komorebi/issues/64 basically the same issue

**Comment by Firefox289** [comment]
> How are you installing it? Are you using the `.deb` package?
> 
> It it's not working in any way, try using our [fork](https://github.com/Komorebi-Fork/komorebi/releases/tag/v2.2.1). If it still doesn't work, please explain every step you're doing to install.

I just install the deb from fork and click it then download in eddy then I open it

**Comment by Ev1lbl0w** [comment]
Try uninstalling this version first, as both versions might be coexisting. Follow the steps in #232.

Run this in a terminal: `where komorebi`, and show me the output it gives.

**Comment by Firefox289** [comment]
ok

**Comment by Firefox289** [comment]
idk how
do sudo apt remove /System/Resources/Komorebi
/System/Applications/
/usr/share/applications/komorebi.desktop
/usr/share/applications/wallpapercreator.desktop


??

**Comment by Firefox289** [comment]
just forget it

i dumb anyways bye

**Comment by T1mors** [comment]
I have the same problem with a static background and no menu with the normal komorebi. I deinstalled it and installed the fork version via the console. When I try to run komorebi I get this error:

~/Downloads/komorebi/build$ komorebi
Welcome to komorebi
Reading config file..
[INFO]: loading Gst

(komorebi:27914): Gdk-ERROR **: 17:25:12.433: The program 'komorebi' received an X Window System error.
This probably reflects a bug in the program.
The error was 'GLXBadDrawable'.
  (Details: serial 661 error_code 160 request_code 152 (GLX) minor_code 29)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the GDK_SYNCHRONIZE environment
   variable to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)
Trace/breakpoint trap (core dumped)

Hopefully this helps to fix my problem :)

**Comment by Ev1lbl0w** [comment]
@T1mors it looks like you're using 2 or more screens. If that's the case, there's currently no solution for [this problem](https://github.com/Komorebi-Fork/komorebi/issues/18), so you'll have to run komorebi on your main screen only with `komorebi --single-screen` *(note that it's only implemented in our fork)*.

**Comment by T1mors** [comment]
Yes, that's it. I am currently using 2 screens. At least the single screen works as intended. Thanks

**Comment by Harishjayabash** [comment]
I have removed the komorebi now the right clicking my screen doesn't give a usaul pop up.

**Comment by pendragons-code** [comment]
https://github.com/cheesecakeufo/komorebi/issues/283

**Comment by pendragons-code** [comment]
https://github.com/cheesecakeufo/komorebi#after-uninstalling-my-desktop-isnt-working-right-blank-or-no-icons

**Comment by GuckicheLorraine** [comment]
Bash: line 1: 404: : command not found
